tests/functional/conftest.py

```diff
@@ -42,10 +42,15 @@
 from PyInstaller import __main__ as pyi_main
 from PyInstaller.utils.cliutils import archive_viewer
 from PyInstaller.compat import is_darwin, is_win, is_py2, safe_repr, \
-  architecture, is_linux
+  architecture, is_linux, suppress
 from PyInstaller.depend.analysis import initialize_modgraph
 from PyInstaller.utils.win32 import winutils
 
+# Monkeypatch the psutil subprocess on Python 2
+if is_py2:
+    import subprocess32
+    psutil.subprocess = subprocess32
```

Comment:
Please state in a comment message why this is required.

Reply:
Python 2.7 doesn't have subprocess.communicate(timeout). subprocess32 does. And since we're only using it for the tests, we don't need to worry about the implications of including this dependency.